test/api/v3/integration/tasks/groups/POST-group_tasks_id_score_direction.test.js

```diff
@@ -48,8 +53,8 @@ describe('POST /tasks/:id/score/:direction', () => {
       user: member.auth.local.username,
       taskName: updatedTask.text,
     }));
+    expect(user.notifications[0].data.groupId).to.equal(guild._id);
```

Comment:
this expectation and the 3 ones above seem to be un-related to the scope of the test that is showing that a task that need approval cannot be scored